_migrations/deploying-migration-assistant/configuration-options.md

```diff
@@ -7,66 +7,35 @@ parent: Deploying migration assistant
 
 # Configuration options
 
-This page outlines the configuration options for three key migrations:
-1. **Metadata Migration**
-2. **Backfill Migration with Reindex-from-Snapshot (RFS)**
-3. **Live Capture Migration with Capture and Replay (C&R)**
+This page outlines the configuration options for three key migrations scenarios:
 
-Each of these migrations may depend on either a snapshot or a capture proxy. The CDK context blocks below are shown as separate context blocks for each migration type for simplicity. If performing multiple migration types, combine these options, as the actual execution of each migration is controlled from the Migration Console.
+1. **Metadata migration**
+2. **Backfill migration with `Reindex-from-Snapshot` (RFS)**
+3. **Live capture migration with Capture and Replay (C&R)**
 
-It also has a section describing how to specify the auth details for the source and target cluster (no auth, basic auth with a username and password, or sigv4 auth).
+Each of these migrations depend on either a snapshot or a capture proxy. They are shown as AWS Cloud Deployment Kits (CDKs). The CDKs on this page are shown as separate blocks for each migration type. If you are performing a migration that suits multiple scenarios, these options can be combined in the migration console.
```

Comment:
```suggestion
Each of these migrations depend on either a snapshot or a capture proxy. There are example cdk.context.json configurations that are used by AWS Cloud Deployment Kits (CDKs) to deploy and configure the Migration Assistant. The example cdk.context.json configurations on this page are shown as separate blocks for each migration type. If you are performing a migration that suits multiple scenarios, these options can be combined.
```

Yea... this is a confusing workflow.  So there is a file called `cdk.context.json` that is used to provided parameters to the CDK tool for deploying Migration Assistant.  Changing the parameters following the recommendations below changes what is deployed and how it is configured.  So I'd call these examples parameter templates or deployment context templates.  After a user configures the cdk.context.json, then they deploy with CDK and then they can start using the MA via the migration console.

Also tweaked that final statement, the migration console is the 'control panel' for all these different migration types and sadly it cannot update the deployed features.